README.md

```diff
@@ -11,9 +11,6 @@ Creating query and executing them are two separate concerns. And thus postgres-q
 - The pool, that allows to connect to the postgres database and execute query.
 - The query builders (insertOne, selectOne, etc..) that allows to generate sql, and the corresponding parameters.
 
-## Documentation
-go to [https://marmelab.com/postgres-queries/](https://marmelab.com/postgres-queries/)
```

Comment:
The doc URL is now on the top of the repo tagline: 
![image](https://user-images.githubusercontent.com/1819833/49521583-0fe92f80-f8a6-11e8-97ec-d351a16e27af.png)